bin/alluxio-start.sh

```diff
@@ -331,33 +343,69 @@ main() {
       sleep 2
       start_worker "${MOPT}"
       start_proxy
+
+      if [[ "${monitor}" ]]; then
+        start_monitor "local"
+      fi
       ;;
     master)
       start_master "${FORMAT}"
+
+      if [[ "${monitor}" ]]; then
+        start_monitor "master"
+      fi
       ;;
     masters)
       start_masters
+
+      if [[ "${monitor}" ]]; then
+        start_monitor "masters"
+      fi
       ;;
     proxy)
       start_proxy
+
+      if [[ "${monitor}" ]]; then
+        start_monitor "proxy"
+      fi
       ;;
     proxies)
       start_proxies
+
+      if [[ "${monitor}" ]]; then
+        start_monitor "proxies"
+      fi
       ;;
     restart_worker)
       restart_worker
+
+      if [[ "${monitor}" ]]; then
+        start_monitor "worker"
+      fi
       ;;
     restart_workers)
       restart_workers
+
+      if [[ "${monitor}" ]]; then
+        start_monitor "workers"
+      fi
       ;;
     safe)
       run_safe
       ;;
     worker)
       start_worker "${MOPT}"
+
+      if [[ "${monitor}" ]]; then
+        start_monitor "worker"
+      fi
       ;;
     workers)
       start_workers "${MOPT}"
+
+      if [[ "${monitor}" ]]; then
+        start_monitor "workers"
```

Comment:
I see this way we don't need to repeat the monitor flag check in each option, I will update the code. Thanks!

Reply:
Just noticed you mentioned logserver, this mean I need to add a new monitor for this process, right?

Reply:
I wouldn't worry about it for now. It isn't included as a part of `start all`, and it doesn't have a thrift RPC server like the rest of the servers, so it will be more difficult to health check.

Reply:
Now you brought up the serving testing topic, I wanted to ask you, currently I am testing the proxy service availability not by RPC, but checking the connection socket of the rest api server. It's that correct?

Reply:
The last action the proxy takes during startup is to start serving the REST service, so we should be all set 👍